lib/my_interceptor.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
```

Comment:
This class is unnecessary outside the scope of tests, and it should not appear under `lib`.